### ISSUE TITLE
feechurs of a completely empty theme - extracted from a 10 minute video

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/thisis#3]
In the comments for [thisis  -  an experimental WordPress Full Site Editing theme ( part 1 )](https://studio.youtube.com/video/LCXcGYTGkLc/edit)

I pointed out a number of feechurs I found while playing with the Site editor.
Some of these may be worthy of issues raised either against thisis, or Gutenberg

Note: The video was recorded with Gutenberg 10.4.1, WordPress 5.7.1, on a Windows 10 machine using the Chrome browser. There were no other plugins activated. 
The feechurs were:

- 3:21 It doesn't look quite right. The Site editor doesn't expect nothing (feechur)
- 3:45 Comments on the toggle block inserter changing to an X (feechur )
- 5:04 Medium uses the post excerpt ... which is pretty useless here ( feechur )
- 5:18 Also an unwanted / unexpected Read more... link ( feechur )
- 6:40 Strange things happening. Why is it displaying toolbar icons? ( feechur )
- 6:45 Why is everything else still showing? (feechur )
- 6:50 I want to visit the real site... weird navigation back to the dashboard ( feechur )
- 8:27 WIBNI I could transform post excerpt to post content ( feechur - already raised as a Feature Request )

**Comment by bobbingwide** [comment]
> 3:21 It doesn't look quite right. The Site editor doesn't expect nothing (feechur )

It's better with v0.1.0. Additionally, use More tools & options > Top toolbar in order to work on the top most block.
This doesn't happen in the block editor since the title field appears at the top.

**Comment by bobbingwide** [comment]
> 8:27 WIBNI I could transform post excerpt to post content ( feechur - already raised as a Feature Request )

https://github.com/WordPress/gutenberg/issues/31137

Wouldn't it also be nice if ( WIABNI )  I could transform a selection of post blocks into a Query block.
I'd created a template with post-content, post-date , post-author, post-hierarchical-terms and post-title then realised I needed to wrap them into the Query block. Dragging and dropping the blocks into the Query block was not the easiest.
When I dragged the post-content block it appeared that I was adding a Cover block.
